src/sys/socket/addr.rs

```diff
@@ -702,7 +702,7 @@ pub mod sys_control {
             let addr = sockaddr_ctl {
                 sc_len: mem::size_of::<sockaddr_ctl>() as c_uchar,
                 sc_family: AddressFamily::System as c_uchar,
-                ss_sysaddr: consts::AF_SYS_CONTROL as uint16_t,
```

Comment:
This is defined as a `c_int`, is `sockaddr_ctl` correct or is `libc` wrong on the datatype?